modules/plugin/geopkg/src/test/java/org/geotools/geopkg/GeoPackageTest.java

```diff
@@ -86,12 +58,24 @@
 import org.opengis.parameter.GeneralParameterValue;
 import org.opengis.referencing.FactoryException;
 
-import com.vividsolutions.jts.geom.Coordinate;
-import com.vividsolutions.jts.geom.Envelope;
-import com.vividsolutions.jts.geom.Geometry;
-import com.vividsolutions.jts.geom.GeometryFactory;
-import com.vividsolutions.jts.geom.Point;
-import com.vividsolutions.jts.geom.PrecisionModel;
+import javax.imageio.ImageIO;
+import javax.media.jai.PlanarImage;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.List;
+import java.util.logging.Level;
+
+import static org.junit.Assert.*;
```

Comment:
As per the style guide, please avoid using * imports. (also, looks like there is a bunch of unnecessary import changes in general - that is probably something you can turn off in your IDE)